backend/x11/backend.c

```diff
@@ -613,6 +614,13 @@ struct wlr_backend *wlr_x11_backend_create(struct wl_display *display,
 	x11->display_destroy.notify = handle_display_destroy;
 	wl_display_add_destroy_listener(display, &x11->display_destroy);
 
+	uint8_t zero = 0;
+	xcb_pixmap_t blank = xcb_create_pixmap_from_bitmap_data(
+		x11->xcb, x11->screen->root, &zero, 1, 1, 1, 0, 0, NULL);
+	x11->cursor = xcb_generate_id(x11->xcb);
```

Comment:
Can we cleanup the cursor and pixmap in `backend_destroy`? wlroots is a library, and compositors are free to create/destroy as many backends as they want, so I'd prefer not to leak anything if possible.

Reply:
backend_destroy destroys the connection, which should also free any server-side resources. However the pixmap should be freed earlier, will do.